rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -160,19 +187,28 @@ struct MetaDataBatch
         return this;
     }
     virtual std::shared_ptr<MetaDataBatch> clone()  = 0;
+    void set_metadata_type(MetaDataType metadata_type) { _type = metadata_type; }
+    MetaDataType metadata_type() { return _type; }
     std::vector<int>& get_label_batch() { return _label_id; }
     std::vector<BoundingBoxCords>& get_bb_cords_batch() { return _bb_cords; }
     std::vector<BoundingBoxCords_xcycwh>& get_bb_cords_batch_xcycxwh() { return _bb_cords_xcycwh; }
     std::vector<BoundingBoxLabels>& get_bb_labels_batch() { return _bb_label_ids; }
     ImgSizes & get_img_sizes_batch() { return _img_sizes; }
     JointsDataBatch & get_joints_data_batch() { return _joints_data; }
+    std::vector<MaskCords>& get_mask_cords_batch() { return _mask_cords; }
```

Comment:
is this different than mask_vertices?

Reply:
The polygon and vertices names are used to represent the counts only. Mask data values are represented as mask_coords